NBitcoin/Key.cs

```diff
@@ -407,6 +408,18 @@ public BitcoinSecret GetWif(Network network)
 			return new BitcoinSecret(this, network);
 		}
 
+		public byte[] GetSeedBytes()

```

Comment:
Call it `ToBytes` and follow how I implemented it in `PubKey`.
Your version is super inefficient.

Reply:
Ah sure